### DISCUSSION TITLE
volume selector on web UI (chrome Android)

### DISCUSSION THREAD

**Discussion by somax80** [rern/rAudio#316] [Fixed Issues]
I've an issue with volume control on web UI.
If I tap on a point around che circle , the volume will adjust (up or down) correctly.
But if I try to swype the bullet around the circle, it freeze down.  I've to reload the web page to have the volume control responding.

I have 20210416 version and the problem started when i update to this version.

Anyone have seen this problem?

thanks,
Max

**Comment by AndyFx12** [comment]
Hi somax80,

yes, I had same problem. Latest update solved this issue for me.
Please update.

regards, Andy

**Comment by AndyFx12** [comment]
Update: Issue is still present, especially if you use multiple devices. Most problems are on my phone in portrait mode.

Andy

**Comment by AndyFx12** [comment]
Finally fixed with today's 20210502 Update.

**Reply by somax80** [reply]
with 20210502 release It's ok .
thank you
Max
